level/level.go

```diff
@@ -96,6 +96,31 @@ func AllowNone() Option {
 	return allowed(0)
 }
 
+// Parse allows to setup allowed levels reading the level parameter.
+// If the level parameter doesn't match one of the four allowed values the defaultOption is applied
+//
+// Useful when we wanna set the allowed level reading from flags, environment variable,
+// configuration file, ...
+func Parse(level string) Option {
```

Comment:
Parse can fail, so this function should return an error. A separate function can wrap Parse with a default option in case of failure.

Reply:
And should this be ParseOption?

Reply:
Are you suggesting `func Parse(level string) (Option, error)` or `func Parse(level string) (Value, error)`? The second would mesh with something like `func Allow(v Value) Option` nicely. That new option could also be used like `level.Allow(level.Info())`.

I am modestly concerned about the potential for bike-shedding on exactly what spellings of the levels `Parse` accepts.

Reply:
I'd default to `level string` being normalized via `strings.TrimSpace` and `strings.ToLower`.